packages/hoppscotch-selfhost-web/package.json

```diff
@@ -77,7 +77,7 @@
     "vite-plugin-inspect": "0.7.42",
     "vite-plugin-pages": "0.31.0",
     "vite-plugin-pages-sitemap": "1.6.1",
-    "vite-plugin-pwa": "0.17.3",
+    "vite-plugin-pwa": "0.20.5",
```

Comment:
I noticed breaking changes in the intermediary versions; might have to evaluate once and see the safest version that we can bump to.